adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/MqttProtocolAdapterProperties.java

```diff
@@ -49,13 +58,67 @@ public final int getCommandAckTimeout() {
      * <p>
      * The default value is {@link #DEFAULT_COMMAND_ACK_TIMEOUT}.
      *
+     * @deprecated Use {@link #setSendMessageToDeviceTimeout(long)} ()} instead.
+     *
      * @param timeout The time out in milliseconds.
      * @throws IllegalArgumentException if the timeout is negative.
      */
+    @Deprecated(forRemoval = true)
     public final void setCommandAckTimeout(final int timeout) {
         if (timeout < 0) {
             throw new IllegalArgumentException("timeout must not be negative");
         }
         this.commandAckTimeout = timeout;
     }
+
+    /**
+     * Gets the waiting for acknowledgement time out in milliseconds for commands published with QoS 1.
+     * <p>
+     * This time out is used by the MQTT adapter for commands published with QoS 1. If there is no acknowledgement
+     * within this time limit, then the command is settled with the <em>released</em> outcome.
+     * <p>
+     * The default value of this property is {@link #DEFAULT_SEND_MESSAGE_TO_DEVICE_TIMEOUT}.
+     *
+     * @return The time out in milliseconds.
+     */
+    public long getSendMessageToDeviceTimeout() {
+        return sendMessageToDeviceTimeout;
+    }
+
+    /**
+     * Sets the waiting for acknowledgement time out in milliseconds for commands published with QoS 1.
+     * <p>
+     * This time out is used by the MQTT adapter for commands published with QoS 1. If there is no acknowledgement
+     * within this time limit, then the command is settled with the <em>released</em> outcome.
+     * <p>
+     * The default value of this property is {@link #DEFAULT_SEND_MESSAGE_TO_DEVICE_TIMEOUT}.
+     *
+     * @param sendMessageToDeviceTimeout The time out in milliseconds.
+     * @throws IllegalArgumentException if the timeout is negative.
+     */
+    public void setSendMessageToDeviceTimeout(final long sendMessageToDeviceTimeout) {
+        if (sendMessageToDeviceTimeout < 0) {
```

Comment:
I took a look at `VertxBasedAmqpProtocolAdapter`. Since we do support a value of 0 there and intend to have an abstraction over protocols I think it makes sense to implement the same behavior for MQTT as you suggested 👍